extensions/vertx-http/deployment/src/main/java/io/quarkus/vertx/http/deployment/VertxHttpProcessor.java

```diff
@@ -563,4 +564,13 @@ NativeImageFeatureBuildItem Brotli4jFeature(HttpBuildTimeConfig httpBuildTimeCon
         }
         return null;
     }
+
+    @BuildStep
+    NativeAccessBuildItem brotli4jNativeAccess(HttpBuildTimeConfig httpBuildTimeConfig) {
+        if (httpBuildTimeConfig.compressors.isPresent()
+                && httpBuildTimeConfig.compressors.get().stream().anyMatch(s -> s.equalsIgnoreCase("br"))) {
+            return new NativeAccessBuildItem();
```

Comment:
I wonder if we should pass a string here with the reason. That could help when we want to figure out who contributed to this.
We got hit by that several times (now, it might not be necessary in this case, I let you decide).